wordpress-mobile/WordPress-iOS
# Integrate version of editor pod.

Integration of this editor fix: https://github.com/wordpress-mobile/WordPress-Editor-iOS/pull/647

How to test:

 * Start a new post
 * Add a image or video to the post
 * While the uploading is happening use the backspace to remove the media
 * Check if the upload status indicator of the post gets updated correctly.

cc @bummytime @diegoreymendez can you guys have a look?
